.github/workflows/ci.yml

```diff
@@ -30,30 +30,11 @@ jobs:
       - name: Install apt packages
         run: |
           sudo apt-get update
-          sudo apt-get install -y autoconf automake build-essential ccache cmake cpanminus gettext gperf language-pack-tr libtool-bin locales ninja-build pkg-config python3 python3-pip python3-setuptools unzip flake8
```

Comment:
We build third-party for clint, so I'm not sure why all of this needs to be removed, but clearly not all of it is needed since we successfully ran clint.  That also implies some of it may no longer be necessary for the other CI jobs.

For lint in particular, getting rid of the `Setup interpreter packages` step and the cpanminus, language-pack-tr, python3, python3-pip, python3-setuptools, and unzip package installs looks legit.

We should keep the `Build third-party` and cache steps so we reduce the CI cost of those.